### PR TITLE
twelve-days: update to match specification

### LINKED ISSUE #1165: Twelve days is out of sync with specification
The specification for twelve days has been change to correct a punctuation issue.

https://github.com/exercism/problem-specifications/blob/84aa47979df7f526bdcb777cacc2c2cc9647dcaa/exercises/twelve-days/description.md

### PR DESCRIPTION
**Pull request by PatrickMcSweeny**
Closes #1165.

### CODE REVIEW

**Comment by PatrickMcSweeny** [comment]
I updated the example to match the suggestion. I don't know Go so I'm open to whatever suggestions you may have.

File: exercises/twelve-days/example.go [hunk 1]
```diff
@@ -39,9 +39,11 @@ func Verse(i int) string {
 	for from := i; from > 0; from-- {
 		if i != 1 && from == 1 {
 			gifts = fmt.Sprintf("%s, and %s", gifts, verse[wording[from]])
-		} else {
-			gifts = fmt.Sprintf("%s, %s", gifts, verse[wording[from]])
-		}
+		} else if i != from {                                                                                                                   
+      gifts = fmt.Sprintf("%s, %s", gifts, verse[wording[from]])
```
Comment (nywilken) [inline exercises/twelve-days/example.go]: Let's run `go fmt` within the exercise directory to fix indentation issues for your latest changes.  Once formatted, the travis build should go green.

**Comment by PatrickMcSweeny** [comment]
@nywilken Done.

**Comment by ferhatelmas** [comment]
Thanks everybody.

**Pull request merged**